### DISCUSSION TITLE
Usage about Nonlinear Inquality Constraints

### DISCUSSION THREAD

**Discussion by kuoyenlo** [meta-pytorch/botorch#1266] [Q&A] [answered]
At the optimize_acqf >>
botorch.optim.optimize.optimize_acqf(acq_function, bounds, q, num_restarts, raw_samples=None, options=None, inequality_constraints=None, equality_constraints=None, nonlinear_inequality_constraints=None, fixed_features=None, post_processing_func=None, batch_initial_conditions=None, return_best_only=True, sequential=False, **kwargs)

There is a setting option for nonlinear_inequality_constraints, but I cant find any tutorial about how to set the inequalities.
Any suggestion about how to use the nonlinear inequality constraints?

**Comment by Balandat** [comment, marked as answer]
@dme65 do you have some pointers?

@kuoyenlo this functionality is currently experimental  and so we don’t really have much documentation on this at this point.
